lib/diagnostic.js

```diff
@@ -6,14 +6,24 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
-}
+
+const getNormalizedWords = function getNormalizedWords(paragraph) {
+  let words = paragraph.split(' ')
```

Comment:
Looks good, I think you're just missing the part where you get rid of the punctuation